cpp/dolfin/common/TimeLogger.cpp

```diff
@@ -59,17 +59,31 @@ Table TimeLogger::timings(std::set<TimingType> type)
 {
   // Generate log::timing table
   Table table("Summary of timings");
+
+  bool time_wall = (type.find(TimingType::wall) != type.end());
```

Comment:
Are the outer braces necessary?

Reply:
I don't know, but I think it looks better. `a = b != c;` looks a bit odd.

Reply:
Looks ok to me. We do `return a ==b` rather than `return (a ==b)`.